codingTest_김준우/k번째 수.js

```diff
@@ -0,0 +1,19 @@
+function solution(array, commands) {
+  let i;
+  let j;
+  let k;
+  let ArrNum = commands.length;
+  let answer = 0;
+  const arr = new Array();
+  for (let a = 0; a < ArrNum; a++) {
```

Comment:
```suggestion
  for (let i = 0; i < arrNum; i++) {
```
for문에선 보통 i를 선언해서 사용합니다.